core/metabolism.py

```
@@ -0,0 +1,252 @@
+        surplus = 10  # 10% surplus for muscle gain
+        return tdee * (1 + surplus / 100)
+    else:
+        raise ValueError(f"Unknown goal: {goal}")  # noqa: TRY003
+
+
+def calculate_deficit_surplus(current_kcal: float, target_kcal: float) -> Dict[str, float]:
+    """Calculate deficit/surplus information.
+
+    EN: Calculate deficit/surplus information.
+
+    Args:
```

Ruff (0.13.3)

Docstring contains ambiguous `о` (CYRILLIC SMALL LETTER O). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF002)